package.json

```diff
@@ -31,19 +31,24 @@
   "author": "charlie.thomas@lifeway.com",
   "license": "ISC",
   "dependencies": {
+    "axios": "^0.19.2",
     "jsonwebtoken": "^8.5.1",
-    "loglevel": "^1.6.8"
+    "loglevel": "^1.6.8",
+    "object.fromentries": "^2.0.2",
```

Comment:
The Typescript team recommends you use a polyfill for `Object.fromEntries`

Grrr, but 🤷 

https://github.com/microsoft/TypeScript/issues/32803